os/board/rtl8721csm/src/component/common/api/wifi/wifi_conf.c

```diff
@@ -1596,8 +1596,8 @@ static void wifi_ap_sta_assoc_hdl( char* buf, int buf_len, int flags, void* user
 #if defined(CONFIG_PLATFORM_TIZENRT_OS)
 	rtk_reason_t reason;
 	memset(&reason, 0, sizeof(rtk_reason_t));
-	if (strlen(buf) >= 17) {			  // bssid is a 17 character string
-		memcpy(&(reason.bssid), buf, 17); // Exclude null-termination
+	if (buf != NULL) {
+		ndbg("STA MAC address is " MAC_FMT "\n", MAC_ARG(buf));

```

Comment:
I think this is informational log, so how about using `nvdbg` instead of `ndbg`?